src/components/HabitCard/HabitCard.js

```diff
@@ -0,0 +1,56 @@
+import "./HabitCard.css"
+import { useState, useEffect, useContext } from "react"
+import { AppContext } from "../../utils/context"
+// import habit7Uncheck from "../../assets/icons/habit7-uncheck.png"
+
+export default function HabitCard(props) {
+  const [checked, setChecked] = useState(0)
+  const [style, setStyle] = useState({
+    backgroundColor: null,
+    // backgroundImage: "url(../../assets/icons/habit7-uncheck.png)",
+  })
+  const { checkedHabitId, setCheckedHabitId } = useContext(AppContext)
+
+  useEffect(() => {
```

Comment:
I like how you are accomplishing this check/uncheck style change... Hopefully this will be a smooth transition when we have icons.